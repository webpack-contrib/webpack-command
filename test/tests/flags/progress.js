const { apply, test, validate } = require('../../util');

const fixture = 'progress/progress';
const opts = { fixture };

let config;

test('--progress', module, () => {
  it(`should validate`, () => {
    expect(validate(opts)).toEqual(true);
  });

  it(`should apply`, () => {
    config = apply(opts);

    expect(config).toMatchSnapshot();
  });
});
