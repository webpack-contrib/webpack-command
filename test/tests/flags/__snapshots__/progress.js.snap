// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flags > --progress > should apply #0 1`] = `
Object {
  "entry": "<PROJECT_ROOT>/test/fixtures/common/entry-a.js",
  "mode": "development",
  "plugins": Array [
    NamedModulesPlugin {
      "options": Object {},
    },
  ],
  "progress": true,
  "reporter": "<PROJECT_ROOT>/test/fixtures/common/test-reporter.js",
}
`;
